src/com/collection/ArrayList_Student.java

```diff
@@ -0,0 +1,46 @@
+import java.util.ArrayList;
```

Comment:
A couple of issues here:

- This file is missing `package` declaration
- Avoid redundant characters in class names ("_", "@", "$")

Reply:
Intellij always adds package declarations. Something went wrong. 
I will not use redundant characters anymore.